populate_db.sql

```diff
@@ -214,3 +244,13 @@ COPY gnosis_protocol_v2.view_solvers
 FROM '/repo/tests/data/gnosis_protocol_v2.view_solvers.csv'
 DELIMITER ','
 CSV HEADER;
+
+COPY gnosis_protocol_v2."GPv2Settlement_call_settle"
+FROM '/repo/tests/data/gnosis_protocol_v2.GPv2Settlement_call_settle.csv'
+DELIMITER ','
+CSV HEADER;
+
+COPY dune_user_generated.cow_trusted_tokens
+FROM '/repo/tests/data/dune_user_generated.cow_trusted_tokens.csv'
+DELIMITER ','
+CSV HEADER;
```

Comment:
These two tables were missing from the initial seed data. Had to be added here. Note that I discovered a bug when trying to download the CSV content from Dune for the `GPv2Settlement_call_settle` table because it contains json content which doesn't work for their CSV download feature. I had to write a stand alone script to get this. Code here:

```py
import csv
from src.duneapi.api import DuneAPI
from src.duneapi.types import Network, DuneQuery

QUERY = """select 
    call_block_number,
    call_block_time,
    call_success,
    call_tx_hash,
    "clearingPrices",
    contract_address,
    tokens
from gnosis_protocol_v2."GPv2Settlement_call_settle"
where call_block_time between '2022-03-01' and '2022-03-11'
and call_success = true
"""



if __name__ == "__main__":
    dune = DuneAPI.new_from_environment()

    to_csv = dune.fetch(DuneQuery.from_environment(
        raw_sql=QUERY,
        name="Call Settle",
        network=Network.MAINNET,
        parameters=[],
    ))

    keys = to_csv[0].keys()

    with open('gnosis_protocol_v2.GPv2Settlement_call_settle.csv', 'w', newline='') as output_file:
        dict_writer = csv.DictWriter(output_file, keys)
        dict_writer.writeheader()
        dict_writer.writerows(to_csv)
```